src/gcp_scanner/scanner.py

```diff
@@ -26,6 +26,9 @@
 from pathlib import Path
 from typing import List, Tuple, Dict, Optional, Union
 
+# Configure the logging module
+logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
We have it already here: https://github.com/google/gcp_scanner/blob/b8c75d6340ba11b81a0e0a5362056d3157c55801/src/gcp_scanner/scanner.py#L529